activity/src/main/java/com/amazonaws/stepfunctions/cloudformation/activity/ResourceModelUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.amazonaws.stepfunctions.cloudformation.activity;
+
+import com.amazonaws.services.stepfunctions.model.DescribeActivityResult;
+import com.amazonaws.services.stepfunctions.model.Tag;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ResourceModelUtils {
```

Comment:
nit: add class level comment

is this class intended to be instantiated? given that it has only static methods, perhaps create a private constructor if it isn't intended to be instantiated.

Reply:
Added class level comment and private constructor